docs/connection-dialog.md

```diff
@@ -100,3 +100,19 @@ __connectionProperties.js (JDBC only)__
 ## Localize your connector
 
 For information on localizing your connection dialogs, see [Localize Your Connector]({{ site.baseurl }}/docs/localize).
+
+## Clear Connection Cache
+Clearing the connection cache might be a necessary step in the development and testing of the connection. To clear last successful connection cache you need to:
+
+**For Windows**
+
+Using Regedit on Windows delete this folder from the registry: 
+`Computer\HKEY_CURRENT_USER\SOFTWARE\Tableau\Tableau <version>\ConnectionSettings\<connector_class>`
+
+**For Mac** 
+
```

Comment:
Probably worth a note here that this command will remove this for all connectors on Mac. I think plist editors exist, but way out of scope for this.

Reply:
Update the doc to reflect all the cache is removed in mac.